### PR TITLE
docs/security: OpenSSL usage

### PR DESCRIPTION
**Pull request by enkore**

### CODE REVIEW

**Comment by enkore** [comment]
OpenSSL has a ton of feature flags, I bet disabling a host of stuff we never use could reduce binary size a bit (excluding libssl is ~400k, libcrypto is about 2 MB).

E.g. ASN1, BIO, BN (bignum), CMS, unused algorithms (DES, RSA, elliptic curves, key exchanges), OCSP routines, PEM and X509, PKCS#1-9000, ...

**Comment by ThomasWaldmann** [comment]
Well, we usually do not build openssl, so we can't use these feature flags, I guess.
Also, maybe building openssl has its own traps & pitfalls? Just guessing.

1MB more is maybe preferable over building openssl (and maybe introducing bugs / issue just by doing that).

File: scripts/borg.exe.spec [hunk 1]
```diff
@@ -38,3 +40,16 @@ exe = EXE(pyz,
           strip=False,
           upx=True,
           console=True )
+
+if False:
```
Comment (ThomasWaldmann) [inline scripts/borg.exe.spec]: rather introduce some flag variable set at the top of the file than hardcoding False here.

Comment (enkore) [inline scripts/borg.exe.spec]: I don't agree, having the explanation what this is about is quite sufficient in such a rarely touched file. No extra flag is needed.

**Pull request merged**